tests/Issue.py

```diff
@@ -263,3 +264,39 @@ def testCreateReaction(self):
 
         self.assertEqual(reaction.id, 16917472)
         self.assertEqual(reaction.content, "hooray")
+
+    def testGetTimeline(self):
+        expected_events = {'referenced', 'cross-referenced', 'locked', 'unlocked', 'closed', 'assigned', 'commented', 'subscribed', 'labeled'}
+        events = self.issue.get_timeline()
+
+        first = events[0]
+        self.assertEqual(15819975, first.id)
+        self.assertEqual("MDE1OlN1YnNjcmliZWRFdmVudDE1ODE5OTc1", first.node_id)
+        self.assertEqual("https://api.github.com/repos/PyGithub/PyGithub/issues/events/15819975", first.url)
+        self.assertEqual("jacquev6", first.actor.login)
+        self.assertEqual(327146, first.actor.id)
+        self.assertEqual("subscribed", first.event)
+        self.assertIsNone(first.commit_id)
+        self.assertIsNone(first.commit_url)
+
+        for event in events:
+            self.assertIn(event.event, expected_events)
+            self.assertIsNotNone(event.created_at)
+            self.assertIsNotNone(event.actor)
+
+            if event.event == "cross-referenced":
+                # cross-referenced events don't include an event id or node_id
+                self.assertIsNotNone(event.source)
+                self.assertEqual(event.source.type, 'issue')
+            else:
+                self.assertIsNotNone(event.id)
+                self.assertIsNotNone(event.node_id)
+
+                if event.event == 'referenced':
```

Comment:
Are there any referenced events in the timeline? Coverage is saying the next two lines don't fire.

Reply:
There wasn't. I removed the check here. Thanks.